packages/fxa-admin-panel/backstage.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: fxa-admin-panel
```

Comment:
Just double checking that there are no extra considerations needed for admin panel. Currently it is only accessible via a VPN connection and proper ldap group headers. IIRC these headers are added at the ingress, which I don't see a config for here. Just bring this up as a thing to check.

Reply:
I don't think there's anything else to mention here, the main entry point is sufficient for the 'where' it is.